consensus/broadcast/shardChainMessenger_test.go

```diff
@@ -7,9 +7,12 @@ import (
 	"time"
 
 	"github.com/multiversx/mx-chain-core-go/data"
-	"github.com/multiversx/mx-chain-go/consensus"
 	"github.com/stretchr/testify/require"
 
+	"github.com/multiversx/mx-chain-go/consensus"
+	consensus2 "github.com/multiversx/mx-chain-go/testscommon/consensus"
```

Comment:
i suggest to rename this to `commonConsensus`, `testscommonConsensus` or something similar